.travis.yml

```diff
@@ -38,6 +38,9 @@ script:
   - nf-core lint ${TRAVIS_BUILD_DIR}
   # Lint the documentation
   - markdownlint ${TRAVIS_BUILD_DIR} -c ${TRAVIS_BUILD_DIR}/.github/markdownlint.yml
-  # Run the pipeline with the test profile
+  # Run the pipeline with barcoded data and mimimap2
   - nextflow run ${TRAVIS_BUILD_DIR} -profile test,docker --aligner minimap2 -ansi-log false
+  # Run the pipeline with barcoded data and graphmap
   - nextflow run ${TRAVIS_BUILD_DIR} -profile test,docker --aligner graphmap -ansi-log false
+  # Run the pipeline with non-barcoded data and mimimap2
+  - nextflow run ${TRAVIS_BUILD_DIR} -profile test_nobc,docker --aligner minimap2 -ansi-log false
```

Comment:
Just for completeness, is it worth adding the `test_nobc` and `--aligner graphmap` test profile too?

Reply:
Ideally, we only add another test to highlight other different potential failures. The previous test already makes sure the `graphmap` component is working so probably not worth it. However, it may be worth adding a test for `--skip_alignment`. We also have to bear in mind that Travis CI has a time limit so need to make sure we dont have too many tests.

Reply:
A `--skip_alignment` test would be great but I suppose there is already the `CALIBRATION_RUN` in the barcoded dataset that won't undergo alignment - I'll leave it up to you

Reply:
Additional test for `--skip_alignment` should be there now. Ill wait for the tests to pass and then merge this into `dev` and then to `master`.

Reply:
Yes, good point. Thats a nice test to see if the pipeline works without providing a reference sequence. The `CALIBRATION_RUN` sample is actually filtered out independent of the `--skip_alignment` parameter:
https://github.com/nf-core/nanoseq/blob/e3ad61c134954f51e36f5097e15cf3a535f8e1ab/main.nf#L464